addons/project/models/project.py

```diff
@@ -1025,6 +1025,7 @@ def _read_group_personal_stage_type_ids(self, stages, domain, order):
         'res.company', string='Company', compute='_compute_company_id', store=True, readonly=False,
         required=True, copy=True, default=_default_company_id)
     color = fields.Integer(string='Color Index')
+    rating_active = fields.Boolean(string='Project Rating Status', related="project_id.rating_active", readonly=True)
```

Comment:
No need to specify the default value.
```suggestion
    rating_active = fields.Boolean(string='Project Rating Status', related="project_id.rating_active")
```